utilities/similarity/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+## Unreleased
```

Comment:
Are you going to maintain this file going forward? If you are unsure, it's better to not include a changelog for now.